src/style.css

```diff
@@ -0,0 +1,23 @@
+body {
+  margin: 0;
+}
+
+.gray__star {
+  display: inline-block;
+  height: 12px;
+  width: 12px;
+  background-image: url(images/star.svg);
+  margin: 0 2px;
+}
+
+.gold__star {
+  display: inline-block;
+  height: 12px;
+  width: 12px;
+  background-image: url(images/star-active.svg);
+  margin: 0 2px;
+}
```

Comment:
You should have only one class - `stars__star` and using pseudo-classes set active start via CSS, not set different classes for elements in HTML

Reply:
If you need more contex - feel free to ping in you slack channel